docs/make.bat

```diff
@@ -0,0 +1,35 @@
+@ECHO OFF
+
+pushd %~dp0
+
+REM Command file for Sphinx documentation
+
+if "%SPHINXBUILD%" == "" (
+	set SPHINXBUILD=sphinx-build
+)
+set SOURCEDIR=source
+set BUILDDIR=build
+
+if "%1" == "" goto help
+
+%SPHINXBUILD% >NUL 2>NUL
+if errorlevel 9009 (
+	echo.
+	echo.The 'sphinx-build' command was not found. Make sure you have Sphinx

```

Comment:
Ah I see. This answered my previous question. Is there way to add this to pyproject.toml? There may be way to install it using PyPI: https://www.sphinx-doc.org/en/master/usage/installation.html#installation-from-pypi